src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -28,12 +30,28 @@ const PhotoGallery = () => {
   useEffect(() => {
     getAlbum();
   }, [getAlbum]);
+
   return (
-    <PicturesContainer>
-      {photosURL.map((photo, index) => (
-        <Image src={photo} key={index} />
-      ))}
-    </PicturesContainer>
+    <>
+      {
+        photosURL.length ===0 ?
+        (
+          <Loading />
+        ):
+        (
+          <PicturesContainer>
+          {photosURL.map((photo, index) => (
+            <LazyLoadImage
+              height="auto"
+              key={index}
+              src={photo}
+              width="100%" 
+            />
+          ))}
+          </PicturesContainer>
```

Comment:
I can still see the footer right under the loading spinner, can we set the min-height for `PicturesContainer`?

Reply:
addressed. Added a minHeight to the Loading component.